packages/components/organisms/f-registration/test-utils/component-objects/f-registration.component.js

```diff
@@ -1,74 +1,85 @@
+
+    get termsAndConditionsLink () { return $(TERMS_AND_CONDITIONS_LINK); }
+
+    get privacyPolicyLink () { return $(PRIVACY_POLICY_LINK); }
+
+    get cookiesPolicyLink () { return $(COOKIES_POLICY_LINK); }
 
-    get component () { return $(REGISTRATION_COMPONENT) }
-    get createAccountButton () { return $(CREATE_ACCOUNT_BUTTON) }
-    get termsAndConditionsLink () { return $(TERMS_AND_CONDITIONS_LINK) }
-    get privacyPolicyLink () { return $(PRIVACY_POLICY_LINK) }
-    get cookiesPolicyLink () { return $(COOKIES_POLICY_LINK) }
-  
     fields = {
         firstName: {
-            get input () { return $(FIRST_NAME_INPUT) }, 
-            get emptyError () { return $(FIRST_NAME_EMPTY_ERROR) }, 
-            get maxLengthError () { return $(FIRST_NAME_MAX_LENGTH_ERROR) } , 
-            get invalidError () { return $(FIRST_NAME_INVALID_ERROR) }
-        }, 
+            errorMessages: {
+                maxCharacters: 'First name exceeds 50 characters',
+                missing: 'Please include your first name',
+                invalidFormat: 'Your first name can only contain letters, hyphens or apostrophes'
+            },
+            get input () { return $(FIRST_NAME_INPUT); },
+            get errorMessage () { return $(FIRST_NAME_ERROR_MESSAGE).innerText; }
+        },
         lastName: {
-            get input () { return $(LAST_NAME_INPUT) }, 
-            get emptyError () { return $(LAST_NAME_EMPTY_ERROR) }, 
-            get maxLengthError () { return $(LAST_NAME_MAX_LENGTH_ERROR) }, 
-            get invalidError () { return  $(LAST_NAME_INVALID_ERROR) }
-        }, 
+            errorMessages: {
+                maxCharacters: 'Last name exceeds 50 characters',
+                missing: 'Please include your last name',
+                invalidFormat: 'Your last name can only contain letters, hyphens or apostrophes'
+            },
+            get input () { return $(LAST_NAME_INPUT); },
+            get errorMessage () { return $(LAST_NAME_ERROR_MESSAGE).innerText; }
+        },
         email: {
-            get input () { return $(EMAIL_INPUT) }, 
-            get emptyError () { return $(EMAIL_EMPTY_ERROR) }, 
-            get invalidError () { return $(EMAIL_INVALID_ERROR) }, 
-            get existsError () { return $(EMAIL_EXISTS_ERROR) }
-        }, 
+            errorMessages: {
+                invalidFormat: 'Please enter your email address correctly',
+                missing: 'Please enter your email address'
+            },
+            get input () { return $(EMAIL_INPUT); },
+            get errorMessage () { return $(EMAIL_ERROR_MESSAGE).innerText; }
+        },
         password: {
-            get input () { return $(PASSWORD_INPUT) }, 
-            get emptyError () { return $(PASSWORD_EMPTY_ERROR) }
+            errorMessages: {
+                minLength: 'Password is less than four characters',
+                missing: 'Please enter a password'
```

Comment:
It concerns me that we're duplicating text. It might make sense to store it somewhere a variable, together with all the other error messages? Also what about internationalisation?

Reply:
I agree with you and @SachaZvetelman that these need to be in translation files, they weren't before this though and I didn't want to increase the scope to do that.  Are you OK if I create a new ticket to cover it?